Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md

```diff
@@ -235,7 +235,7 @@ Anti-debugging features can be preventive or reactive. As the name implies, prev
 
 ##### Anti-JDWP-Debugging Examples
 
-In the chapter "Reverse Engineering and Tampering," we talked about JDWP, the protocol used for communication between the debugger and the Java Virtual Machine. We showed that it is easy to enable debugging for any app by patching its manifest file, and changing the `ro.debuggable` system property which enables debugging for all apps. Let's look at a few things developers do to detect and disable JDWP debuggers.
+In the chapter "Reverse Engineering and Tampering", we talked about JDWP, the protocol used for communication between the debugger and the Java Virtual Machine. We showed that it is easy to enable debugging for any app by patching its manifest file, and changing the `ro.debuggable` system property which enables debugging for all apps. Let's look at a few things developers do to detect and disable JDWP debuggers.

```

Comment:
odd that git renders everything italic since line 118... Any idea what is going on here?

Reply:
I was looking at it and I really cannot tell why, maybe it's some bug rendering the diff of c-highlighted code (?). The final view looks good actually.

https://github.com/OWASP/owasp-mstg/blob/7f10208145113fe04217ce653f3252bf3b201988/Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md